### ISSUE TITLE
Call Web API always fails with 401

### ISSUE THREAD

**Issue by jasonnutter** [Azure-Samples/active-directory-b2c-javascript-msal-singlepageapp#36]
I'm attempting to run this sample and cannot get it to successfully call the web api, as it returns a 401 error every time.

Repro:
1. Clone repo and run `npm install`.
2. Run the server.
3. Click to login (tried both a personal @outlook.com and @gmail.com account).
4. Click the button to call the web api.
5. Get the following output:

```
User 'Jason Nutter' logged-in
Error calling the Web api:
error
```

In the terminal:
```
jquery-3.2.1.min.js:4 GET https://fabrikamb2chello.azurewebsites.net/hello 401 (Unauthorized)
```

**Comment by bgavrilMS** [comment]
Yes, I can repro every time. Sadly the web api code sample is used as part of "calling your own web api" in other samples as well, so it's a more widespread problem.

**Comment by jennyf19** [comment]
@jasoncoolmax @bgavrilMS 
I didn't realize your samples were broken too, until I tried it today. Our .NET samples and your js samples are all using the same node.js sample to make the api call.

The issue is w/the node.js api (https://fabrikamb2chello.azurewebsites.net/hello), which has hard coded "login.microsoftonline.com". 

I have fixed it, plus a few other things, and can run it on localhost. Next week, @valnav  and I will work to host a new API with the fix for b2clogin.com and we will update all the samples.

Here's my [branch](https://github.com/Azure-Samples/active-directory-b2c-javascript-nodejs-webapi/compare/jennyf/b2clogin) for testing the web api running on localhost, if you want to try it out.

cc: @henrik-me @DarylThayil @jmprieur @negoe
